lib/iris/quickplot.py

```diff
@@ -126,12 +129,17 @@ def _get_titles(u_object, v_object):
     return xlabel, ylabel, title
 
 
-def _label_1d_plot(*args):
+def _label_1d_plot(*args, **kwargs):
```

Comment:
*explicit is better than implicit*: we don't want to catch all kwargs, because we do nothing with them. Instead, explicitly put axes as a keyword argument here. 😄

Reply:
True enough! I'll change that :+1: